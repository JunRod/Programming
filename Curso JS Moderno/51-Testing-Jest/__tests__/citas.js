import Citas from "../js/classes/Citas";

describe("Probar la clase de Citas", () => {
    const citas = new Citas()

    test("Agregar una nueva cita", () => {
        const citaObj = {
            mascota: 'Perrillo',
            propietario: 'Juniior',
            telefono: '02323233',
            fecha: '10-30-2022',
            hora:'18:04',
            sintomas: 'solo duerme'
        }

        citaObj.id = Date.now();

        citas.agregarCita(citaObj)

        //Prueba
        expect(citas).toMatchSnapshot()
    })
})