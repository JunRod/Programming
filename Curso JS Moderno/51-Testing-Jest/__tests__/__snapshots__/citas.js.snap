// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Probar la clase de Citas Agregar una nueva cita 1`] = `
Citas {
  "citas": [
    {
      "fecha": "10-30-2022",
      "hora": "18:04",
      "id": 1669133441547,
      "mascota": "Perrillo",
      "propietario": "Juniior",
      "sintomas": "solo duerme",
      "telefono": "02323233",
    },
  ],
}
`;
